torch/_dynamo/output_graph.py

```diff
@@ -443,11 +443,14 @@ def get_backward_state_proxy(self):
         if self.backward_state_proxy is None:
             if self.export:
                 unimplemented("backward_state does not support export")
+            example_value = BackwardState()
```

Comment:
Can you check (add a test?) that this doesn't cause a warning? Direct initialization of FunctionCtx usually induces a warniing